examples/mri/cartesian_reconstruction.py

```diff
@@ -2,137 +2,96 @@
 Neuroimaging cartesian reconstruction
 =====================================
 
-Credit: A Grigis, L Elgueddari, H Carrie
+Author: Chaithya G R
 
 In this tutorial we will reconstruct an MRI image from the sparse kspace
-measurments.
+measurements.
 
 Import neuroimaging data
 ------------------------
 
 We use the toy datasets available in pysap, more specifically a 2D brain slice
-and the acquistion cartesian scheme.
-We also add some gaussian noise in the image space.
+and the cartesian acquisition scheme.
 """
 
 # Package import
-import pysap
+from modopt.math.metrics import ssim
```

Comment:
But, I think this is required to understand and scale the performance levels of the reconstruction with respect to baseline. @zaccharieramzi and @philouc to comment

Reply:
@chaithyagr if this is indeed "required" then you will have to open an issue on ModOpt before this PR can be merged.

Reply:
Is it just a matter of opening an issue in Modopt? Or is it a matter of getting a PR for Modopt also? 
If it is a PR, is it just a few lines of code? In that case, we can proceed this way, else I could modify the code to get the ssim from Scikit-Image.

Reply:
It would require a PR. a) to make Scikit-Image a required dependency (I don't think the examples should include optional dependencies), b) to fix the usage of `compare_ssim` for v0.16, which is different from v0.15.

Reply:
Thanks for this thread. On the user view, SSIM might be really useful to quantifiy image quality in the end. Nonetheless, if this means adding a new dependency to scikit-image just for this computation, what about put the SSIM computation in comments and just report the pSNR?

Reply:
@philouc that would certainly be a simpler solution.

Reply:
How about I just directly use scikit image's compare_ssim function? We would not be dependent on this package for pysap or Modopt, but if you want to run the example, you need SciKit-Image

Reply:
> SSIM computation in comments and just report the pSNR?
I am fine with this also, both approaches seem good to me

Reply:
@chaithyagr I would say either make a PR to ModOpt or use the PSNR. I think having external dependencies for examples is a bit odd.

Reply:
Hmmm, okay, let me just wait for @zaccharieramzi and @LElgueddari to chime in so that we are in sync on what goes for the examples

Reply:
> It would require a PR. a) to make Scikit-Image a required dependency (I don't think the examples should include optional dependencies), b) to fix the usage of `compare_ssim` for v0.16, which is different from v0.15.

b) resolved in ModOpt [PR 67](https://github.com/CEA-COSMIC/ModOpt/pull/67).

Reply:
Ill add a comment for scikit-Image dependency and also add the same in README. I think this is the best way forward

Reply:
Since pysap is an image reconstruction package, having reliable metrics for comparison purposes is mandatory, so in my point of view, it is important to have the SSIM score there. 
So one thing that we can do (to avoid adding a required dependency on the skimage for modopt) is to install modopt within pysap-mri but with optional dependencies.

Reply:
I do agree with Loubna on this. I will try to see if I can install Modopt with optional dependencies. 
Worse case, to keep things moving, I will add Scikit-Image as a dependency on pysap-mri (which makes sense also, given we have an image reconstruction package)

Reply:
This is ongoing in a PR. @sfarrens I dont see any changes need to this PR, can you merge it if you feel the same? We are trying to fix the pyNFFT install by picking up from github. That PR holds addition for SciKitImage also.